content/post/91-ipfs-dev-meetings-2018.md

```diff
@@ -0,0 +1,62 @@
+- [Lightning Demos - IPFS Graphs - Juan Benet](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=20)
+- [Lightning Demos - IPFS & p2p - Steven Allen](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=21)
+- [Lightning Demos - Private Networks GO and JS-IPFS - Jacob Heun](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=22)
+- [Lightning Demos - IPFS-SENC - Juan Benet](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=23)
+- [Lightning Demos - Data Security in IPFS - Dr. Ian Preston](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=24)
+- [Lightning Demos - Rabin - Juan Benet](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=25)
+- [Lightning Demos - Service Worker Gateway - Vasco Santos](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=26)
+- [Lightning Demos - QRI - Brendan O' Brien](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=27)
+- [Lightning Demos - OpenBazaar - Chris Pacia](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=28)
+- [Lightning Demos - Peer Bandwidth Chart - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=29)
+- [Lightning Demos - MFS Fuse - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=30)
+- [Lightning Demos - Peer Maps - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=31)
+- [Lightning Demos - Textile - Andrew Hill](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=32)
+- [Lightning Demos - pServer - Wei Yang](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=33)
+- [Lightning Demos - window.IPFS - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=34)
+- [Lightning Demos - IPLD Explorer CLI - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=35)
+- [Lightning Demos - Files and IPFS Companion - Marcin Rataj](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=36)
+- [Lightning Demos - VRQ Recovery - jonnycrunch](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=37)
+- [Project Updates, Lightning Talks Q&A - Decentralized Identifiers - jonnycrunch](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=38)
+- [Live Keynotes - Peer Discovery Protocols - Juan Benet](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=39)
```

Comment:
Something is weird with these urls. I just tried to open all of them in Firefox, Firefox Incognito and Chrome and they always take me to Hector's talk, although I can see that the Index is different. Does anyone have the same issue?

Reply:
@momack2 seems that something is weird with the urls. @olizilla and I are directed to Hector's talk for all of them.

Reply:
Same issue here. Let me troubleshoot - I'm cleaning up all of the dev meetings videos on the backend today and will find a solution.

Reply:
These used to work for me - but also break now (I just see Matt's talk over and over). Zak - I think it honestly just has to grab these full URLs from the playlist (https://www.youtube.com/playlist?list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr)